shared/actions/chat.js

```diff
@@ -332,6 +392,19 @@ function _threadToPagination (thread) {
   }
 }
 
+function _maybeAddTimestamp (message: Message, prevMessage: Message): MaybeTimestamp {
+  if (message.type !== 'Text' || prevMessage.type !== 'Text') {
```

Comment:
> We should likely always do the timestamp, not just between text messages

In this case:

prevMessage = Timestamp
Message = Text

We wouldn't want to add another timestamp to make Timestamp => Timestamp => Text.  I'll change the test to be `if (message.type === 'Timestamp' || prevMessage.type === 'Timestamp') { return null }` here.